package.json

```diff
@@ -69,8 +69,8 @@
     "@griffel/webpack-loader": "2.2.10",
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33498&commitId=58466f7ee42cc8e39457c807ddc786da0e208090&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|6| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|12| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay RTL.chromium.png|8| Changed|
|Drawer.Full Overlay High Contrast.chromium.png|5191| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>